docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6347,10 +6347,13 @@ It defines the following fields
 
 `GenericData` is an addition to p4data.proto- which aims at adding support for
 data types which aren't covered by P4Data. This incorporates all the data types
-used as primitive data types like bytes, float, strings. It contains P
+used as primitive data types like bytes, float, strings. It contains p4type as
+well. P4type is to be used for any data type present in P4. If there is a
+conflict, like bool, then the P4 data type should be used.
 
   * bitstring : Bytes type used for simple unsigned integers or bytes
-  * float : Floating type values
+  * float : Floating type values. Target might return a lower-precision
+    ceiled value even if field is non-volatile.
```

Comment:
How do we reconcile R/W symmetry?

Reply:
How about a precision length field in p4info? The controller will know about length and will have expectations set as to what will the reading value be

Reply:
This probably requires some group discussion, this is s tricky issue. Precision length might not capture all the details (round-up/round-down, etc.)